.teamcity/src/main/kotlin/common/Os.kt

```diff
@@ -66,9 +66,10 @@ enum class Os(
     val jprofilerHome: String,
     val killAllGradleProcesses: String,
     val perfTestWorkingDir: String = "%teamcity.build.checkoutDir%",
-    val perfTestJavaVendor: String = "oracle",
+    val perfTestJavaVendor: JvmVendor = JvmVendor.oracle,
     val buildJavaVersion: JvmVersion = JvmVersion.java11,
-    val perfTestJavaVersion: JvmVersion = JvmVersion.java8
+    val perfTestJavaVersion: JvmVersion = JvmVersion.java8,
```

Comment:
🤔 Would it make sense to add the Java version/vendor to the constructor of `PerformanceTestCoverage` instead of setting it per operating system? This way, we could also add performance tests for different Java versions easily.

Reply:
Makes sense, but let's do this in a separate PR.